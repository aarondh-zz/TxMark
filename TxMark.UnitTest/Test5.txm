
(set: $fubar to 3.14159 * 42 / 2)

 fubar = (print: $fubar) 

fubar = (print: $fubar) and don't you forget it!

