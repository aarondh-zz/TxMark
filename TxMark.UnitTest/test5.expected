<p>

</p>
fubar = 65.97339
fubar = 65.97339 and don&#39;t you forget it! 